src/hotspot/cpu/riscv/templateInterpreterGenerator_riscv.cpp

```diff
@@ -781,17 +764,27 @@ void TemplateInterpreterGenerator::generate_fixed_frame(bool native_call) {
 
   // set sender sp
   // leave last_sp as null
-  __ sd(x30, Address(sp, 9 * wordSize));
+  __ sd(x19_sender_sp, Address(sp, 9 * wordSize));
   __ sd(zr, Address(sp, 8 * wordSize));
 
-  // Move SP out of the way
+  // Get mirror
+  __ load_mirror(t2, xmethod);
   if (!native_call) {
     __ ld(t0, Address(xmethod, Method::const_offset()));
     __ lhu(t0, Address(t0, ConstMethod::max_stack_offset()));
-    __ add(t0, t0, frame::interpreter_frame_monitor_size() + 2);
+    __ add(t0, t0, MAX2(3, Method::extra_stack_entries()));
     __ slli(t0, t0, 3);
     __ sub(t0, sp, t0);
-    __ andi(sp, t0, -16);
+    __ andi(t0, t0, -16);
+    // Store extender SP and mirror
```

Comment:
typo here: should be "extended" instead of "extender".

Reply:
Fixed.